### ISSUE TITLE
Implement downloading of log files

### ISSUE THREAD

**Issue by carlspring** [strongbox/strongbox#1698]
# Task Description

There are currently two types of log files:
* Actively being written to (`*.log`)
* Archived (`*.gz)

It should be possible to download log files. We need to have a download link button/icon.

If a `*.log` file is clicked, it should be streamed.

If a `*.gz` file is clicked, it should be downloaded.

# Tasks

The following tasks will need to be carried out:
* [ ] Make it possible to download log files via the UI.
  * [ ] Add download link.
  * [ ] When a `*.log` file is clicked from the browse list, stream it.
* [ ] Create a test case.

# Help

* [Our chat](https://chat.carlspring.org/)
* Points of contact:
  * @steve-todorov
  * @sbespalov

**Comment by carlspring** [comment]
This task is currently blocked by another issue.

@steve-todorov : Could you please create a task for that and link it back as a dependency, thanks? :)

**Comment by despoinasr** [comment]
I am interested in working on this issue when it's unblocked.

**Comment by steve-todorov** [comment]
This task is unfortunately now obsolete and I'll mark it as won't fix. The better part of it has been fixed within #1704. The streaming part is something we will be skipping for now as we will explore a few ways to actually achieve it.

**Comment by carlspring** [comment]
@despoinasr ,

Thanks for your interest! Please, join our [chat channel](https://chat.carlspring.org/), so we could discuss what tasks you might be able to help out with.
